subsys/partition_manager/pm.yml.fmfu

```diff
@@ -0,0 +1,9 @@
+#include <autoconf.h>
+
+fmfu_storage:
+  placement: {before: [end]}
+#ifdef CONFIG_SPI_NOR
+  align: CONFIG_SPI_NOR_FLASH_LAYOUT_PAGE_SIZE
```

Comment:
Is it guaranteed that SPI NOR driver is used for external flash if this option is set?

Reply:
i am not sure it is guaranteed.
i do not know the best way to handle this item, as i am not too familiar with this subject.
this is how it was implemented in https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/partition_manager/pm.yml.modem_trace